Line delimited JSON doesn't work as CLI input file

<!--
# Instructions

Please follow the following instructions so we can help you best.

## Before filing an issue

1. Make sure there isn't already an issue present, i.e. you can search the issues.
2. Make sure it's not an issue with your data. Check the validity of your JSON using something like http://jsonlint.com
3. Make sure you're using the latest version of json2csv.
4. Know what's wrong? Submit a Pull Request instead with a failing test and/or a fix.

## Filing an issue

1. Include the command or code you used.
2. Include a sample dataset that we can test against.
3. Include your output/error.
-->

An error occurs when providing a line delimited JSON file as input to the CLI application. This is because the file is being imported with `require`, which will only work with a valid JSON document. As a workaround, piping the data through stdin works as expected.

```
➜ json2csv -L -i example.jsonl
module.js:529
    throw err;
    ^

Error: Cannot find module '/home/user/example.jsonl'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:476:23)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at getInput (/home/user/.config/yarn/global/node_modules/json2csv/bin/json2csv.js:58:13)
    at /home/user/.config/yarn/global/node_modules/json2csv/bin/json2csv.js:104:3
    at getFields (/home/user/.config/yarn/global/node_modules/json2csv/bin/json2csv.js:49:5)
    at Object.<anonymous> (/home/user/.config/yarn/global/node_modules/json2csv/bin/json2csv.js:99:1)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
```

--- a/bin/json2csv.js
+++ b/bin/json2csv.js
@@ -52,16 +52,28 @@ function getFields(callback) {
 }
 
 function getInput(callback) {
-  var input, isAbsolute, rows;
+  var input = '';
 
   if (program.input) {
-    isAbsolute = isAbsolutePath(program.input);
-    input = require(isAbsolute ? program.input : path.join(process.cwd(), program.input));
+    var isAbsolute = isAbsolutePath(program.input);
+    var inputPath = isAbsolute ? program.input : path.join(process.cwd(), program.input);
 
+    if (program.ldjson) {
+      fs.readFile(inputPath, 'utf8', function (err, data) {
+        if (err) {
+          return callback(err);
+        }
+
+        input = parseLdJson(data);
+        callback(null, input);
+      });
+      return;
+    }
+
+    input = require(inputPath);
     return callback(null, input);
   }
 
-  input = '';
   process.stdin.resume();
   process.stdin.setEncoding('utf8');
 
@@ -72,11 +84,9 @@ function getInput(callback) {
     debug('Could not read from stdin', err);
   });
   process.stdin.on('end', function () {
-    if (program.ldjson) {
-      rows = parseLdJson(input);
-    } else {
-      rows = JSON.parse(input);
-    }
+    var rows = program.ldjson
+      ? parseLdJson(input)
+      : JSON.parse(input);
 
     callback(null, rows);
   });

